src/models/rfcProps.js

```diff
@@ -290,6 +290,7 @@ const properties = {
 			{ id: 'YOUTUBE', name: 'YouTube', placeholder: 'https://youtube.com/…' },
 			{ id: 'MASTODON', name: 'Mastodon', placeholder: 'https://mastodon.social/…' },
 			{ id: 'DIASPORA', name: 'Diaspora', placeholder: 'https://joindiaspora.com/…' },
+			{ id: 'NEXTCLOUD', name: 'Nextcloud', placeholder: 'https://nextcloud.example.com/…' },
```

Comment:
For me it wasn't obvious that the profile link should be put here. Maybe add a hint to the placeholder, e.g. `Link to profile page (https://nextcloud.example.com/…)`.

Reply:
thank you both for reviewing. working on your suggestions